articles/tokens/access_token.md

```diff
@@ -1,50 +1,142 @@
 ---
```

Comment:
@cabralmartin Please review this doc. All changes was based on this input https://forum.it.auth0.com/t/corrections-to-access-token-documentation/50